### ISSUE TITLE
Review codestyle rules

### ISSUE THREAD

**Issue by nickbattam** [willrogers/cssgen#49]
The landscape codestyle check is running with `veryhigh` strictness - this is generating a large number of warnings and error, some of which we are happy to carry.
For these case the rules should be updated so the reported errors are things that we want to resolve.

Rules generating false-positives can be fine-tuned, see https://docs.landscape.io/advanced.html

**Comment by nickbattam** [comment]
I attempted to use a #pragma to disable specific rules in the code but this resulted in Landscape failing and I reverted the commit (f335fb19)

**Comment by willrogers** [comment]
I think `.pylintrc` is probably the way to go, although it is a bit of overhead.

**Comment by nickbattam** [comment]
I agree, that way the same rules will be used on any locally run style checking. It's not too large an overhead - we can fine tune the rules we're using as and when there's an issue.
